### ISSUE TITLE
test/tst_*.c programs incorrectly setting dimids

### ISSUE THREAD

**Issue by edwardhartnett** [ccr/ccr#94]
These codes were all copied from the original tst_bzip2.c, which had a bug.

Both dimension ids are being assigned to dimid[0]. Whoops!

```
       /* Create dims. */
        if (nc_def_dim(ncid, X_NAME, NX, &dimid[0]))
	  ;
        if (nc_def_dim(ncid, Y_NAME, NY, &dimid[0]))
	  ;

        /* Create the variable. */
        if (nc_def_var(ncid, VAR_NAME, NC_INT, NDIM2, dimid, &varid)) ERR;

```

Instead, this should be:

```
        /* Create dims. */
        if (nc_def_dim(ncid, X_NAME, NX, &dimid[0])) ERR;
        if (nc_def_dim(ncid, Y_NAME, NY, &dimid[1])) ERR;

        /* Create the variable. */
        if (nc_def_var(ncid, VAR_NAME, NC_INT, NDIM2, dimid, &varid)) ERR;

```

**Comment by czender** [comment]
Good catch!

**Comment by edwardhartnett** [comment]
Caused errors on the mac but oddly not linux platforms.

**Comment by edwardhartnett** [comment]
This was fixed in #93
